Error: the backend never received a keepalive from the agent

```
--- FAIL: TestCheckSubdue (18.43s)
	process_test.go:204: the backend never received a keepalive from the agent
{"component":"agent","error":"Connection error: websocket: close 1006 (abnormal closure): unexpected EOF","level":"error","msg":"transport receive error","time":"2018-02-09T05:31:42-06:00"}
{"component":"agent","error":"Connection closed: the websocket connection is no longer open","level":"error","msg":"transport receive error","time":"2018-02-09T05:31:42-06:00"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x1503e5b]

goroutine 31 [running]:
github.com/sensu/sensu-go/agent.(*Agent).receivePump(0xc42010e800, 0x1a6d4e0, 0xc42018cc40)
	/Users/nattea/go/src/github.com/sensu/sensu-go/agent/agent.go:413 +0x1cb
created by github.com/sensu/sensu-go/agent.(*Agent).Run
	/Users/nattea/go/src/github.com/sensu/sensu-go/agent/agent.go:518 +0x33f
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -40,6 +40,7 @@ subcommand.
 be executed.
   be executed.
 - Fixed a bug in the HTTP API where resource names could not contain special characters.
+- Resolved a bug in the keepalive monitor timer which was causing it to erroneously expire.
 
 ## [2.0.0-alpha.17] - 2018-02-13
 ### Added

--- a/backend/monitor/monitor.go
+++ b/backend/monitor/monitor.go
@@ -93,7 +93,7 @@ func (m *Monitor) start() {
 		return
 	}
 
-	timerDuration := m.Timeout * time.Second
+	timerDuration := m.Timeout
 	m.timer = time.NewTimer(timerDuration)
 	m.resetChan = make(chan time.Duration)
 	go func() {

